doc/topics/packaging/index.rst

```diff
@@ -23,11 +91,40 @@ You only need to run rpmbuild in the Salt repo:
 How to build deb packages
 =========================
 
-You only need to add a symlink and run debuild in the Salt repo:
+Install the dependencies:
+
+.. code-block:: bash
+
+   # apt install -y python3 python3-venv python3-pip build-essential devscripts debhelper bash-completion git
+
+If you want to build for a specific Salt version, you will need to install tools and changelog dependencies:
```

Comment:
**Suggestion (phrasing, non-blocking):** Same comment as above. Instead of "If you want to build..." maybe just say "To build..."